CHANGELOG.md

```diff
@@ -1,10 +1,32 @@
 # Changelog
 
-## [7.1.1] - 12.11.2021
+## [8.0.0] - 02.03.2022
 
 ## Changed
```

Comment:
Perhaps we should add a "Breaking changes" section as well? I can imagine most are wondering what is warranting a major (and not just a minor) version increment.

Reply:
That is a very good suggestion. We will fix that right away 👍